linchpin/provision/roles/openstack/tasks/provision_os_object.yml

```diff
@@ -40,7 +40,7 @@
 - name: "Async:: provision/deprovision swift containers when count specified"
   os_object:
     auth: "{{ auth_var | omit_filter(omit) }}"
-    container: "{{ container.1 }}_{{ container.4 }}"
+    container: "{{ container.1 }}default_delimiter{{ container.4 }}"
```

Comment:
@skatlapa According to jinja template syntax the default_delimiter should be in {{  }}
else it will be taken as a string. 
Have you tested this piece?